Tasks/UseRubyVersionV0/package.json

```diff
@@ -26,6 +26,7 @@
     "@types/node": "^6.0.101",
     "@types/q": "^1.0.7",
     "azure-pipelines-task-lib": "^2.9.3",
+    "azure-pipelines-tasks-utility-common": "^2.0.0",
     "vsts-task-tool-lib": "^0.5.1"
```

Comment:
Could you please change it to `"azure-pipelines-tool-lib": "^0.12.0"` and update `package-lock.json` and import statements also?

Reply:
@egor-bryzgalov done